### ISSUE TITLE
(Bug / Improvement) hardcoded text height

### ISSUE THREAD

**Issue by danilogr** [MartinFk/VRQuestionnaireToolkit#21]
It seems that the location of the buttons, grids, and pretty much everything is hard-coded to questions with a single line of text.
For example, I tried to add the full NASA TLX question to the NASA TLX questionnaire that comes with VRQuestionnaireToolkit and this is what I got:
![image](https://user-images.githubusercontent.com/1302815/118712450-a63f1080-b7d5-11eb-918b-63cac2ca3b14.png)

As looked further into the source code, it seems that all content-generation behaviours rely on a hard-coded value for the text height:
![image](https://user-images.githubusercontent.com/1302815/118712687-ed2d0600-b7d5-11eb-8cb5-ab386d99f1d2.png)

Is there an easy way to fix this? I have hard-coded a longer length per question to solve my problem, but I think that it would be great if the authors of this toolkit made it flexible enough to support multi-line questions. This is especially important for validation because the real NASA TLX has longer text descriptions that *matter*. I.e., the shortened NASA TLX  in use by this toolkit hasn't been validated.

**Comment by MartinFk** [comment]
I completely agree, this is kind of a legacy issue. When I originally developed the toolkit for myself, I had a pretty hard time dealing with Unity3D´s dynamic layouting system. There were many non-trivial issues which I could not sufficiently resolve in time. Eventually I gave up and just worked with fixed offsets. All standard question types should support at least(!) two-line questions (except the NASA, because the layout was specifically generated for it, as mentioned in the documentation). 

That being said, it has been on our list for quite some time, and we plan to release a more dynamic UI layout with version 2.0. However, this unfortunately requires some major changes, and will most likely not happen before the end of summer.

Thank you very much for bringing it up, your feedback helps us to prioritize tasks and improve the toolkit.
